### PR TITLE
sane-backends: fix system LibreSSL usage

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

---

Fix a couple of issues highlighted in https://github.com/Homebrew/homebrew-core/issues/1957.

### CODE REVIEW

File: Formula/net-snmp.rb [hunk 1]
```diff
@@ -25,12 +25,13 @@ def install
       --with-defaults
       --with-persistent-directory=#{var}/db/net-snmp
       --with-logfile=#{var}/log/snmpd.log
-      --with-mib-modules=host ucd-snmp/diskio
```
Comment (zmwangx) [inline Formula/net-snmp.rb]: What about `--with-mib-modules=host\ ucd-snmp/diskio`?

Comment (ilovezfs) [inline Formula/net-snmp.rb]:
```
9:45 PM <ilovezfs_> tdsmith: going to backslash?
9:45 PM <ilovezfs_> or just << separately
9:46 PM <tdsmith> i figured the latter; does a backslash work
9:46 PM <ilovezfs_> irb(main):001:0> %W[a b c\ d e f]
9:46 PM <ilovezfs_> => ["a", "b", "c d", "e", "f"]
9:46 PM <ilovezfs_> seems to
9:46 PM <tdsmith> so it does
9:46 PM <tdsmith> backslash it is
```

Comment (DomT4) [inline Formula/net-snmp.rb]: backslash looks a bit weird, but can't be bothered to argue. Pushed.

Comment (tdsmith) [inline Formula/net-snmp.rb]: applied to master in deaf31b, sorry

**Comment by mistydemeo** [comment]
:thumbsup:

File: Formula/net-snmp.rb (hunk 1 above)
Comment (DomT4) [inline Formula/net-snmp.rb]: I just want to go and sleep 😓

**Comment by DomT4** [comment]
Off to bed for a bit. Feel free to merge this when green if you like, or if people get impatient, I'm not fussed about preserving my GPG signature in the main tree for `core` stuff.

Pull bottles since the revision on one got bumped, and if you're feeling especially lovely grabbing Sierra bottles after via the testing VM would be ❤️ x 💯.

**Pull request closed without merging**